front-end/src/pages/IndividualShow.js

```diff
@@ -219,8 +219,7 @@ const IndividualShow = ({ id, type }) => {
             <div className="show-details">
               <form id="show-form" onSubmit={handleSubmit}>
                 <h3 id="title">{show.title}</h3>
-                <Link to="/my-shows/17">
-                  {/*TODO: Change this to link to logged in user's my-shows*/}
+                <Link to={`/my-shows/${loggedInUser.id}`}>
```

Comment:
More of a note for later, once the all shows page is implemented, but since a non-logged in user can access an individual show's page (to just look at it), the back button should (if a user is not logged in), redirect to the all shows page.